Solutions/Marain.Operations.Specs/Integration/Bindings/ContainerSetupBindings.cs

```
@@ -78,7 +87,7 @@ public static Task TaskSetupOperationsControlTenants(FeatureContext featureConte
         [BeforeScenario]
         public static void SetupScenario(FeatureContext featureContext)
         {
-            FakeOperationsRepository repository = ContainerBindings.GetServiceProvider(featureContext).GetService<FakeOperationsRepository>();
+            FakeOperationsRepository? repository = ContainerBindings.GetServiceProvider(featureContext).GetService<FakeOperationsRepository>();
             repository?.Reset();
```

Shouldn't that use `!` rather than `?`. I.e. should it blow up if there's no repo present, rather than silently failing?

---

Good point. Fixed